src/content/migrate/5.md

```diff
@@ -124,25 +137,55 @@ Using `/* webpackChunkName: '...' */`: Make sure to understand the intention:
-
-- You can hide deprecation warnings by running node with `--no-deprecation` flag, e.g.: `node --no-deprecation node_modules/webpack/bin/webpack.js`. This should only be a temporary workaround.
-- Plugins and Loaders contributors can follow the advises in the deprecation messages to improve the code.
+- Level 5: Runtime Errors. This is tricky. You probably have to debug to find the problem. A general advise is difficult here.
+- Level 6: Deprecation Warnings. You probably get a lot of deprecation warnings. This is not directly a problem. Plugins need time to catch up with core changes. Please report these deprecations to the plugins. These deprecations are only warnings and the build will still work with only minor drawbacks (like less performance).
+- Level 7: Performance issues. Usually performance should improve with webpack 5, but there are also a few cases where performance get worse.
+
+- Regarding Runtime Errors:
+    - `process` is not defined.
+        - Webpack 5 do no longer include a polyfill for this Node.js variable. Avoid using it in frontend code.
+        - Want to support frontend and browser usage? Use the `exports` or `imports` package.json field to use different code depending on environment.
+            - To support older bundlers use the `browser` field too.
+            - Alternative: Wrap code blocks with `typeof process` checks. Note that this is worse regarding bundle size.
+        - Want to use environment variables with `process.env.VARIABLE`? You need to use the `DefinePlugin` or `EnvironmentPlugin` to define these variables in the configuration.
+            - Consider using `VARIABLE` instead and make sure to check `typeof VARIABLE !== "undefined"` too. `process.env` is Node.js specific and should be avoided in frontend code.
+- Regarding Deprecation Warnings:
+    - You can hide deprecation warnings by running node with `--no-deprecation` flag, e.g.: `node --no-deprecation node_modules/webpack/bin/webpack.js`. This should only be a temporary workaround.
+    - Plugins and Loaders contributors can follow the advises in the deprecation messages to improve the code.
+- Regarding Performance issues:
+    - Profile where the time is spend.
+        - `--profile --progress` displays a very simple performance profile now
+        - `node --inspect-brk node_modules/webpack/bin/webpack.js` + [`chrome://inspect`](chrome://inspect) / [`edge://inspect`](edge://inspect) (see profiler tab).
+            - You can save these profiles to files and provide them in issues.
+            - Add `--no-turbo-inlining` for better stack traces in some cases
+    - Time for building modules in incremental builds can be improved by reverting to unsafe caching like in webpack 4:
+        - `module.unsafeCache: true`
+        - But this might affect the ability to handle some of the changes to the code base
```

Comment:
This needs to be clarified to the reader: "some of the changes to the code base" is abstract. Does it mean during missing updates during HMR?
```suggestion
        - But this might affect the ability to handle some of the changes to the code base
```